apps/dcs-admin/src/app/features/user-profile/user-profile.component.scss

```diff
@@ -9,36 +9,49 @@
     min-height: 80vh;
 
     .avatar-content {
+      padding: map_get($spacings, 4);
+
       .avatar {
         --size: 15rem;
         border-radius: 50%;
         overflow: hidden;
         padding: 0;
-        max-width: var(--size);
-        max-height: var(--size);
+        width: var(--size);
+        height: var(--size);
+        border: 1px solid map_get($md-primary, 100);
 
         a {
           cursor: pointer;
 
           .icon-hover {
             opacity: 0;
             position: absolute;
-            min-height: var(--size);
-            min-width: var(--size);
+            max-height: var(--size);
+            max-width: var(--size);
+            height: 100%;
+            width: 100%;
             background-color: rgba(#000, .3);
             color: rgba(#fff, .5);
             border-radius: 50%;
             font-size: calc(var(--size) / 3);
-            transition: opacity .4s;
-
-            &:hover {
-              opacity: 1;
-            }
+            transition: opacity map_get($time, 2);
+            z-index: 100;
           }
 
           img {
             width: 100%;
             height: auto;
+            transition: opacity map_get($time, 2);
+          }
+
+          &:hover {
+            img {
+              filter: saturate(0%);
```

Comment:
If it is 0 you don't need to use the percent, with this should be enough `saturate(0)`